File: Web1/ClientApp/my-angular-app/src/app/Client/models/tintuc.ts
export interface Tintuc {
    tintucId: number;                // Mã tin tức
    tieuDe: string;                  // Tiêu đề của tin tức
    hinhAnh: string;                 // Đường dẫn hình ảnh
    moTaNgan: string;                // Mô tả ngắn
    ngayDang: string;                // Ngày đăng (chuỗi)
    ngayCapNhat: string;             // Ngày cập nhật (chuỗi)
    luongKhachTruyCap: number;      // Số lượng khách truy cập
    soLuongComment: number;          // Số lượng bình luận
    danhmucId: number;               // ID danh mục
    binhLuans: any[];                // Danh sách bình luận (có thể định nghĩa thêm nếu cần)
    danhmuc: any;                    // Danh mục (có thể định nghĩa thêm nếu cần)
    tenDanhMuc?: string;
    trangThai?: boolean;
    noiDung?: string;
  }
  

File: Web1/Data/TinTuc.cs
using System;
using System.Collections.Generic;
using Web1.Data;

namespace Web1.Data;

public partial class TinTuc
{
    public int TintucId { get; set; }

    public string TieuDe { get; set; } = null!;

    public string? HinhAnh { get; set; }

    public string? MoTaNgan { get; set; }

    public DateTime? NgayDang { get; set; }

    public DateTime? NgayCapNhat { get; set; }

    public int? LuongKhachTruyCap { get; set; }

    public bool? TrangThai { get; set; }

    public string? NoiDung { get; set; }

    public int? DanhmucId { get; set; }

    public virtual ICollection<BinhLuan> BinhLuans { get; set; } = new List<BinhLuan>();

    public virtual DanhMuc? Danhmuc { get; set; }

}
